constructor/shar.py

```diff
@@ -92,6 +92,7 @@ def create(info, verbose=False):
         for cf in os.listdir(cache_dir):
             if cf.endswith(".json"):
                 p_t.add(join(cache_dir, cf), 'pkgs/cache/' + cf)
+    p_t.add(join(tmp_dir, 'conda-meta', 'history'), 'conda-meta/history')
```

Comment:
Decided to do this at installer create time rather than install time because, if we did it at install time, we'd have to carry some of this information around in another data structure somehow.  Doing it at installer create time is just easier.